src/components/sidebar/left-sidebar/TotalStats.tsx

```diff
@@ -1,31 +1,30 @@
-import React, { useContext } from "react";
+import React, { useContext, useEffect, useState } from "react";
 import { AppContext } from "../../../context";
-import { aggregateRecords } from "../../../metaAnalysis";
 import InformationIcon from "../../shared/InformationIcon";
 import { MIN_DENOMINATOR } from "../../../metaAnalysis"
 import './TotalStats.css';
 import Translate from "../../../utils/translate/translateService";
+import httpClient from "../../../httpClient";
 
 
 export default function TotalStats() {
   const [state] = useContext(AppContext);
+  const [countries, setNumCountries] = useState(0);
+  const [seroprevalence, setSeroprevalence] = useState<any>(null);
+  const [n, setN] = useState(0);
 
-  // Factor in "include_in_n" for population unfiltered geography estimates
-  const included_records = state.filters.population_group.size === 0 ? state.filtered_records.filter(r => r.include_in_n) : state.filtered_records;
-
-  const { seroprevalence, n } = aggregateRecords(included_records);
-  function onlyUnique(value: any, index: number, self: any) {
-    return self.indexOf(value) === index && value !== null;
-  }
-  const countryDict = included_records
-    .map(o => {
-      if (o.seroprevalence !== null && o.denominator !== null) {
-        return o.country
-      }
-      return null;
-    })
-    .filter(onlyUnique);
-  const countries = countryDict.length;
+  useEffect(() => {
+    if(state.filtered_records.length > 0){

```

Comment:
Is it smarter to constantly check inputs that we pass or the inputs that you guys receive on the backend? As a service I feel like it's smarter to sanitize your inputs rather than expect perfection

Reply:
We actually probably don't need to do this check for state.filtered_records.length > 0

Reply:
Gotcha, okay